File: csharp/src/IO.Swagger/Model/Address.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Text.RegularExpressions;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Runtime.Serialization;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.ComponentModel.DataAnnotations;
using SwaggerDateConverter = IO.Swagger.Client.SwaggerDateConverter;

namespace IO.Swagger.Model
{
    [DataContract]
    public partial class Address :  IEquatable<Address>, IValidatableObject
    {
        public Address(string addressLine1 = default(string), string addressLine2 = default(string), string city = default(string), string state = default(string), string zipCode = default(string))
        {
            this.AddressLine1 = addressLine1;
            this.AddressLine2 = addressLine2;
            this.City = city;
            this.State = state;
            this.ZipCode = zipCode;
        }

        [DataMember(Name="AddressLine1", EmitDefaultValue=false)]
        public string AddressLine1 { get; set; }

        [DataMember(Name="AddressLine2", EmitDefaultValue=false)]
        public string AddressLine2 { get; set; }

        [DataMember(Name="City", EmitDefaultValue=false)]
        public string City { get; set; }

        [DataMember(Name="State", EmitDefaultValue=false)]
        public string State { get; set; }

        [DataMember(Name="ZipCode", EmitDefaultValue=false)]
        public string ZipCode { get; set; }
    }
}

File: CodeSamples/nodejs/src/model/Address.js
import ApiClient from '../ApiClient';

export default class Address {
    constructor() {
        this.AddressLine1 = '';
        this.AddressLine2 = '';
        this.City = '';
        this.State = '';
        this.ZipCode = '';
    }

    static constructFromObject(data, obj) {
        if (data) {
            obj = obj || new Address();

            if (Object.prototype.hasOwnProperty.call(data, 'AddressLine1')) {
                obj['AddressLine1'] = ApiClient.convertToType(data['AddressLine1'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'AddressLine2')) {
                obj['AddressLine2'] = ApiClient.convertToType(data['AddressLine2'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'City')) {
                obj['City'] = ApiClient.convertToType(data['City'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'State')) {
                obj['State'] = ApiClient.convertToType(data['State'], 'String');
            }
            if (Object.prototype.hasOwnProperty.call(data, 'ZipCode')) {
                obj['ZipCode'] = ApiClient.convertToType(data['ZipCode'], 'String');
            }
        }
        return obj;
    }
}
